homeassistant/components/vicare/climate.py

```diff
@@ -73,6 +77,12 @@
 VICARE_TEMP_HEATING_MIN = 3
 VICARE_TEMP_HEATING_MAX = 37
 
+VICARE_HEATING_CURVE_SLOPE_MIN = 0.3
+VICARE_HEATING_CURVE_SLOPE_MAX = 3.5
+
+VICARE_HEATING_CURVE_SHIFT_MIN = -13
+VICARE_HEATING_CURVE_SHIFT_MAX = 40
```

Comment:
Could/Should these be two number entities instead?

Reply:
Still i think a number entity is what makes sense here. Disabled by default.

Reply:
We have a marker for such things, which is the entity category. The entity category can be set to be a config for these specific entities. This makes them available from the UI (on the device page only), and excludes them from dashboards, selectors, voice assistants and such things. 

Yet, if a user still has a use case, they can use them with all of the previously mentioned if added by the user explicitly. 

The use of entities, in general, is preferred over services.